src/Module/Admin/Site.php

```diff
@@ -467,7 +468,10 @@ protected function content(array $request = []): string
 			'$show_help'        => ['show_help', DI::l10n()->t('Show help entry from navigation menu'), !DI::config()->get('system', 'hide_help'), DI::l10n()->t('Displays the menu entry for the Help pages from the navigation menu. It is always accessible by calling /help directly.')],
 			'$singleuser'       => ['singleuser', DI::l10n()->t('Single user instance'), DI::config()->get('system', 'singleuser', '---'), DI::l10n()->t('Make this instance multi-user or single-user for the named user'), $user_names],
 
-			'$maximagesize'     => ['maximagesize', DI::l10n()->t('Maximum image size'), DI::config()->get('system', 'maximagesize'), DI::l10n()->t('Maximum size in bytes of uploaded images. Default is 0, which means no limits. Be aware that this setting does not affect server-side upload limits.')],
+			'$maximagesize'     => ['maximagesize', DI::l10n()->t('Maximum image size'), DI::config()->get('system', 'maximagesize'), DI::l10n()->t('Maximum size in bytes of uploaded images. Default is 0, which means no limits.') .
+													DI::l10n()->t('The value of <code>upload_max_filesize</code> in your <code>PHP.ini</code> needs be set to at least the desired limit. ') .
+													DI::l10n()->t('Currently <code>upload_max_filesize</code> is set to ' . Strings::getBytesFromShorthand(ini_get('upload_max_filesize')) . ' (' . ini_get('upload_max_filesize') . 'B). '),
```

Comment:
Due to the way gettext work, we can't have string concatenation inside a `t()` call. This is why it's including a `sprintf()` call where the dynamic arguments are added after the translation string in the `t()` call.
```suggestion
			'$maximagesize'     => ['maximagesize', DI::l10n()->t('Maximum image size'), DI::config()->get('system', 'maximagesize'), DI::l10n()->t('Maximum size in bytes of uploaded images. Default is 0, which means no limits.
													The value of <code>upload_max_filesize</code> in your <code>PHP.ini</code> needs be set to at least the desired limit.
													Currently <code>upload_max_filesize</code> is set to %s (%sB)', Strings::getBytesFromShorthand(ini_get('upload_max_filesize'), ini_get('upload_max_filesize')),
```

Reply:
And once more I learned something. Thank you very much for directly addressing it!